nemo_skills/pipeline/train.py

```diff
@@ -160,7 +160,9 @@ def train(
     disable_wandb: bool = typer.Option(False, help="Disable wandb logging"),
     with_sandbox: bool = typer.Option(False, help="If sandbox is required for code generation"),
     partition: str = typer.Option(None, help="Specify partition for jobs"),
-    average_steps: list[int] = typer.Option(None, help="List of checkpoint steps to average"),
+    average_steps: str = typer.Option(
```

Comment:
Can you please add a check that if there is a " " inside average_steps we will raise an error and ask to check if it's formatted properly with the comma? Not really needed long-term, but to make it easier for people to transition from the previous way (which did work through python api), so that they don't just get their training jobs failing but instead see an error at the submission time

And if you can do the same for benchmarks to cover https://github.com/Kipok/NeMo-Skills/issues/153, that would be great